### ISSUE TITLE
Scaffold with faker-js

### ISSUE THREAD

**Issue by darryldaniel** [fluffynuts/newts#18]
I recently ran `npx newts` and chose to include faker. It installs that weird version (6.6.6). Can we replace it with `faker-js` please?

**Comment by fluffynuts** [comment]
yeah, I've been meaning to do a bit of an update in newts - faker-js is just part of it. Please remind me tho - I'm likely to forget!

**Comment by fluffynuts** [comment]
@darryldaniel I actually started on this, but coming back to it today, I realise why I didn't complete it - there's another issue, upstream with expect-more-jest: https://github.com/JamieMason/expect-more/issues/57 which blocks a clean generate-then-test run anyway. It's not a blocker, but I'd like to solve that as well.
